src/interceptors/fetch/index.ts

```diff
@@ -46,13 +47,21 @@ export class FetchInterceptor extends Interceptor<HttpRequestEventMap> {
 
       this.logger.info('awaiting for the mocked response...')
 
+      const signal = interactiveRequest.signal
+      const requestAbortRejection = new DeferredPromise<string>()
+
+      signal.addEventListener('abort', () => requestAbortRejection.reject())
```

Comment:
Should we forward the abort information exposed to this listener, like abort reason and such? It does work without it so I assume the abort controller already exposes that information in the abort error, is that correct? 

As in, why this is not needed?

```js
signal.addEventListener('abort', () => {
  requestAbortRejection.reject(signal.reason)
})
```

🤔

Reply:
I understand how it affects the promise flow, us using the abort reason as the rejection value and that it may produce false negatives, but my question was about whether we have to respect that reason in any way? As in, forward it directly. We can always reject with an object, or a different data structure to prevent false positives.